File: CS/CustomControlInGrid/CustomRepositoryItem/IExport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CustomControlInGrid {
    public interface IExport {
        DevExpress.XtraPrinting.IVisualBrick GetBrick(DevExpress.XtraEditors.PrintCellHelperInfo info);
    }
}

File: VB/CustomControlInGrid/CustomRepositoryItem/IExport.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace CustomControlInGrid
	Public Interface IExport
		Function GetBrick(ByVal info As DevExpress.XtraEditors.PrintCellHelperInfo) As DevExpress.XtraPrinting.IVisualBrick
	End Interface
End Namespace
